### PR TITLE
Favorite function fix

### PR DESCRIPTION
**Pull request by nori3506**
Modify Favorite function

### CODE REVIEW

**Comment by nori3506** [comment]
Even navigate to somewhere, display contents and db condition are super fine.

File: client/src/features/product.js [hunk 1]
```diff
@@ -242,13 +242,15 @@ const productSlice = createSlice({
     [saveProduct.rejected]: (state) => finishLoading(state),
     [saveProduct.fulfilled]: (state, action) => {
       state.savedProducts.push(action.payload)
+      updateProduct(state, action.payload)
       state.loading = false
```
Comment (satoshinbp) [inline client/src/features/product.js:246]:
```suggestion

```

File: client/src/features/product.js [hunk 2]
```diff
@@ -242,13 +242,15 @@ const productSlice = createSlice({
...
     },
 
     [unsaveProduct.pending]: (state) => startLoading(state),
     [unsaveProduct.rejected]: (state) => finishLoading(state),
     [unsaveProduct.fulfilled]: (state, action) => {
       state.savedProducts = state.savedProducts.filter((product) => product._id !== action.payload._id)
+      updateProduct(state, action.payload)
       state.loading = false
```
Comment (satoshinbp) [inline client/src/features/product.js:254]:
```suggestion

```

**Review by satoshinbp — CHANGES_REQUESTED** [review]

**Comment by nori3506** [comment]
Modified! thanks!

**Pull request merged**